fvm/env.go

```diff
@@ -612,6 +619,51 @@ func (e *hostEnv) RemoveAccountKey(address runtime.Address, index int) (publicKe
 	return e.transactionEnv.RemoveAccountKey(address, index)
 }
 
+func (e *hostEnv) AddAccountKey(
+	address runtime.Address,
+	publicKey *runtime.PublicKey,
+	hashAlgo runtime.HashAlgorithm,
+	weight int,
+) (*runtime.AccountKey, error) {
+
+	if e.isTraceable() {
+		sp := e.ctx.Tracer.StartSpanFromParent(e.transactionEnv.traceSpan, trace.FVMEnvAddAccountKey)
+		defer sp.Finish()
+	}
+
+	if e.transactionEnv == nil {
+		return nil, errors.New("adding account keys is not supported")
+	}
+
+	return e.transactionEnv.AddAccountKey(address, publicKey, hashAlgo, weight)
+}
+
+func (e *hostEnv) GetAccountKey(address runtime.Address, index int) (*runtime.AccountKey, error) {
+	if e.isTraceable() {
+		sp := e.ctx.Tracer.StartSpanFromParent(e.transactionEnv.traceSpan, trace.FVMEnvGetAccountKey)
+		defer sp.Finish()
+	}
+
+	if e.transactionEnv == nil {
+		return nil, errors.New("getting account keys is not supported")
+	}
```

Comment:
is it possible an script need to get account key ?

Reply:
Yes, that would be great. How could the accounts (`Accounts`) be accessed in a script?

Reply:
I was mostly asking the question, I was not sure if scripts can call getKeys